### DISCUSSION TITLE
[FEATURE]: add a landing page

### DISCUSSION THREAD

**Discussion by Dun-sin** [Dun-sin/Whisper#340] [Ideas]
### Description

The [Landing page](whischat.vercel.app)  is just two buttons, which are the login buttons.

It can't be empty like that, there needs to be information about the project.

### Additional information

If you want to work on this, please do a design in figma first. 
edit: AI gave some good design, so you don't need to make a design anymore

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

None

[a comment by an automated account is not shown]

**Comment by Dun-sin** [comment]
> Hey Dun-sin, I would like to work on this issue.

sure, please provide your proposed design

**Comment by github-harshit** [comment]
Hey Dun-sin , I would like to work on this , I have crafted a go-to design, and I invite you to request any potential adjustments or enhancements.    
https://www.figma.com/file/mSIjiAzJ60Yl8EUwtNIfCT/Whisper-Design?type=design&node-id=0%3A1&mode=design&t=HqlUDk7VtIyAs6CP-1

**Comment by Dun-sin** [comment]
> Hey Dun-sin , I would like to work on this , I have crafted a go-to design, and I invite you to request any potential adjustments or enhancements. https://www.figma.com/file/mSIjiAzJ60Yl8EUwtNIfCT/Whisper-Design?type=design&node-id=0%3A1&mode=design&t=HqlUDk7VtIyAs6CP-1

sure, can you give me permission to access the file on figma

**Comment by github-harshit** [comment]
Can you check now

**Comment by Dun-sin** [comment]
> Can you check now

sent a request on figma

**Comment by Dun-sin** [comment]
> here's my submission: Whischat Landing Page design https://www.figma.com/file/FrVfWVsQAD9uxQulm4SGsB/whischat-%7BLanding-Page%7D?type=design&node-id=0%3A1&mode=design&t=TA0IctRSfBcscYKn-1

hi @github-harshit is more simple and easy to read, you can have a look at his if you wish and you can request to collab,feel free

**Comment by mathiasayivor** [comment]
> here's my submission: Whischat Landing Page design
> https://www.figma.com/file/FrVfWVsQAD9uxQulm4SGsB/whischat-%7BLanding-Page%7D?type=design&node-id=0%3A1&mode=design&t=TA0IctRSfBcscYKn-1

Is it just me? But this looks like its a company rather than a fun app

**Comment by Dun-sin** [comment]
> > here's my submission: Whischat Landing Page design
> > https://www.figma.com/file/FrVfWVsQAD9uxQulm4SGsB/whischat-%7BLanding-Page%7D?type=design&node-id=0%3A1&mode=design&t=TA0IctRSfBcscYKn-1
> 
> Is it just me? But this looks like its a company rather than a fun app

yeah, it's too clustered

**Comment by MaheshRautrao** [comment]
Hi @Dun-sin I want to work on this.
Here is my proposed design.
Please provide feedback so I can start working on it.

https://www.figma.com/file/POwLhDMKh3gUHK6aC2ZZvc/Whisper?type=design&node-id=0%3A1&mode=design&t=Tf20AqBy6iH02gJU-1

**Comment by Dun-sin** [comment]
> Can you check now 

Hi I left some comments, I don't think you got it?🤔

**Comment by dhananjay-Byte** [comment]
@Dun-sin I would like to work on the landing page request. below is my design for the landing page. I want to design as minimal as possible.

![whisper landing page](https://github.com/Dun-sin/Whisper/assets/82323909/d247194c-98e1-49c3-b80d-1ff0c0c241d3)

**Comment by Dun-sin** [comment]
> > Can you check now
> 
> Hi I left some comments, I don't think you got it?🤔

@github-harshit

**Comment by samcasmmm** [comment]
@Dun-sin review this ? [Figma File Link](https://www.figma.com/file/gofUiwJTH5fyGMyKupTTY4/Untitled?type=design&node-id=0%3A1&mode=design&t=NH7Q0ijwUnP8FJGQ-1)

**Comment by Dun-sin** [comment]
> @Dun-sin I would like to work on the landing page request. below is my design for the landing page. I want to design as minimal as possible.
> 
> ![whisper landing page](https://user-images.githubusercontent.com/82323909/271790120-d247194c-98e1-49c3-b80d-1ff0c0c241d3.png)

this isn't good, doesn't follow any good UI/UX principle. kudos to you for trying tho

**Comment by Dun-sin** [comment]
> @Dun-sin review this ? [Figma File Link](https://www.figma.com/file/gofUiwJTH5fyGMyKupTTY4/Untitled?type=design&node-id=0%3A1&mode=design&t=NH7Q0ijwUnP8FJGQ-1)

that's not a landing page, so it doesn't count, good work tho, but I'm afraid you wasted your time, that's why you should always read the issue and its description. Keep that in mind for next time.

**Comment by Dun-sin** [comment]
AI gave some pretty good designs, we just need to twerk things a little, which would you vote for:
A
![image](https://github.com/Dun-sin/Whisper/assets/78784850/769453c9-63d1-4eff-9b3f-7ca874046504)

B
![image](https://github.com/Dun-sin/Whisper/assets/78784850/540cb17a-1ed4-4927-9f55-4ddaf3186d53)

C
![image](https://github.com/Dun-sin/Whisper/assets/78784850/abac5bde-d2e5-4028-8152-6e816315b045)

D
![image](https://github.com/Dun-sin/Whisper/assets/78784850/f5e64dec-90cf-424d-9c8d-d88dc1872d5c)

**Comment by zabihshaik** [comment]
I think D looks good, and also matches / suits the earlier design better. @Dun-sin

**Reply by Dun-sin** [reply]
i agree
